README.md

```diff
@@ -1,26 +1,42 @@
-# Portfolio-Desktop
+# Portfolio
 
-> My Portfolio website that help others understand who i am and what technologies i know and how to get in touch with me
+> In this project I created a pertfolio which is adaptable on Desktop as well as on mobile/smartphone. My Portfolio website help others understand who i am and what technologies i know and how to get in touch with me. Please view my portfolio page in your browser by following this link https://hadi111jafari.github.io/Portfolio/
 
 
 ## Built With
 
 - html
 - css
 
+## Getting Started
+1. Please go to https://hadi111jafari.github.io/Portfolio/
+2. Click on repositories
+3. Choose repository "Portfolio" or click to  https://hadi111jafari.github.io/Portfolio/
+4. Then clone the repository to your local machine by using the following steps:
+     1. From repository click on code button and copy clone cammand(Kindly make sure you choose SHH format)
+     2. From your terminal type "git clone" command line
+     3. On same line paste the the copied clone command from GitHub
+     4.  Then click Enter. Now you are the repository/directory on your local machine.
+     
+### Setup
+GitHub account 
```

Comment:
- [ ] I recommend deleting this section, because you are already providing setup instructions  in Getting started section

Reply:
resolved